api/v1alpha1/consumer_types.go

```diff
@@ -106,19 +111,22 @@ type ConsumerStatus struct {
 }
 
 type InstanceState struct {
+	// LastScaleEvent  *metav1.Time `json:"lastScaleEvent,omitempty"`
```

Comment:
Why commented?

Reply:
this is a part of future work on scaling, I'll remove it for now